docs/configuration/multiple-delivery-clients.md

```diff
@@ -44,13 +136,30 @@ public class Startup
 
     public void ConfigureServices(IServiceCollection services)
     {
-        services.AddDeliveryClient("projectA", Configuration, "ProjectAOptions", NamedServiceProviderType.Autofac);
-        services.AddDeliveryClient("projectB", Configuration, "ProjectBOptions", NamedServiceProviderType.Autofac);
+        services.AddMultipleDeliveryClientFactory
+        (
+            factoryBuilder => factoryBuilder
+                .AddDeliveryClient
+                (
+                    "projectA",
+                    _ =>
+                    {
+                        var options = new DeliveryOptions();
+                        config.Configuration.GetSection("MultipleDeliveryOptions:ProjectA")
+                            .Bind(options);
+                        return options;
+                    },
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectAProvider())
+                )
```

Comment:
It seems that
`.Build()`
is missed here

Reply:
ae78d43